ui/src/components/Profile.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div class="q-pa-md row items-start">
+    <q-card class="full-width profile" flat bordered>
+      <q-img
+        src="https://cdn.quasar.dev/img/parallax2.jpg"
+        :style="{ maxHeight: '15em' }"
+      />
+      <q-card-section class="q-mb-none __section">
+        <q-avatar class="__avatar">
+          <img
+            v-if="!profileImageBlob"
+            :src="
+              value.image
+                ? value.image
+                : 'https://cdn.quasar.dev/img/avatar.png'
+            "
+            class="cursor-pointer __image"
+            @click="handleImageChange"
+          />
+          <img
+            :src="profileImageBlob"
+            class="cursor-pointer __image"
+            @click="handleImageChange"
+          />
+        </q-avatar>
+        <q-file class="hidden" ref="imageUpload" v-model="profileImage" />
+        <q-card-section class="q-mb-none q-pa-xs q-pr-lg">
+          <div class="text-h5 q-mt-sm q-mb-md q-gutter-md">
+            <q-input
+              label="Username"
+              borderless
+              class="text-h5"
+              :value="value.username"
+              :mask="profileKeys['username'].mask || '@xxxxxxxxxx'"
+              @input="(val) => inputHandler(val, 'username')"
+            />
+            <q-input
+              v-if="profileKeys['bio']"
+              label="Bio"
+              autogrow
+              filled
+              class="text-caption text-grey"
+              :value="value.bio"
+              @input="(val) => inputHandler(val, 'bio')"
+            />
+          </div>
+          <div class="row q-col-gutter-md">
+            <template v-for="item in Object.keys(profileKeys)">
+              <div
+                :key="item"
+                :class="
+                  profileKeys[item].classes
+                    ? profileKeys[item].classes
+                    : 'col-lg-4 col-md-6 col-12'
+                "
+                v-if="item !== 'bio' && item !== 'username'"
+              >
+                <q-input
+                  :label="profileKeys[item].label"
+                  :value="value[profileKeys[item].name]"
+                  :disable="profileKeys[item].disable"
+                  :autogrow="profileKeys[item].autogrow"
+                  :mask="profileKeys[item].mask"
+                  @input="(val) => inputHandler(val, item)"
+                />
```

Comment:
Two problems here
1. you are iterating an object converted from `this.schema`-- however, in JS, properties in object is unordered. To keep orders in schema, we should iterate through original list(`this.schema`).
2. we should avoid using v-for and v-if https://eslint.vuejs.org/rules/no-use-v-if-with-v-for.html

```suggestion
            <template v-for="item in profileFields">
              <div
                :key="item.name"
                :class="
                  item.classes
                    ? item.classes
                    : 'col-lg-4 col-md-6 col-12'
                "
              >
                <q-input
                  :label="item.label"
                  :value="value[item.name]"
                  :disable="item.disable"
                  :autogrow="item.autogrow"
                  :mask="item.mask"
                  @input="(val) => inputHandler(val, item)"
                />
              </div>
```

Don't forget to add:

```
computed: {
  // ...
   profileFields() {
      return this.schema.filter(item => item.name !== 'bio' && item.name !== 'username')
    }
}
```